udata/tests/api/test_datasets_api.py

```diff
@@ -96,6 +96,28 @@ def test_dataset_api_sorting(self):
         self.assertEqual(len(response.json['data']), 3)
         self.assertEqual(response.json['data'][0]['id'], str(to_follow.id))
 
+    def test_dataset_api_sorting_created(self):
+        user = self.login()
+        first = VisibleDatasetFactory(title="first created dataset")
+        second = VisibleDatasetFactory(title="second created dataset")
+        response = self.get(url_for('api.datasets', sort='created'))
+        self.assert200(response)
+        self.assertEqual(response.json['data'][0]['id'], str(first.id))
+
+        response = self.get(url_for('api.datasets', sort='-created'))
+        self.assert200(response)
+        self.assertEqual(response.json['data'][1]['id'], str(first.id))
```

Comment:
I would find it more understandable the other way around (as you did with last_modified)
```suggestion
        self.assertEqual(response.json['data'][0]['id'], str(second.id))
```